sync/sync-store/src/main/java/com/duckduckgo/sync/store/model/SyncDatabaseModels.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.sync.store.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import org.threeten.bp.Instant
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.format.DateTimeFormatter
+
+@Entity(
+    tableName = "sync_attempts",
+)
+data class SyncAttempt(
+    @PrimaryKey(autoGenerate = true) val id: Int = 0,
+    val timestamp: String = getUtcIsoLocalDate(),
+    val state: SyncState,
+    val meta: String = ""
```

Comment:
what do we expect to have in `meta`?

Reply:
I’m adding comments now, just for testing purposes. Might get rid of it